modules/custom/falcon_commerce/falcon_commerce.install

```diff
@@ -0,0 +1,68 @@
+<?php
+
+/**
+ * @file
+ * Install, update and uninstall functions for Falcon Commerce.
+ */
+
+use Drupal\commerce_store\Entity\Store;
+use Drupal\commerce_price\Price;
+use Drupal\commerce_product\Entity\ProductVariation;
+use Drupal\commerce_product\Entity\Product;
+
+/**
+ * Implements hook_install().
+ */
+function falcon_commerce_install() {
+  drupal_flush_all_caches();
+
+  // Create default Commerce Store.
+  // The store's address.
+  $address = [
+    'country_code' => 'US',
+    'address_line1' => '8 The Green',
+    'address_line2' => 'Ste A',
+    'locality' => 'Dover',
+    'administrative_area' => 'DE',
+    'postal_code' => '19901',
+  ];
+
+  // The currency code.
+  $currency = 'EUR';
+
+  // Import the currency.
+  $currency_importer = \Drupal::service('commerce_price.currency_importer');
+  $currency_importer->import($currency);
```

Comment:
Why do we need to import the currency if there is a config `commerce_price.commerce_currency.EUR.yml` for it?

Reply:
When using the `drush config-import` command the module installation takes place before the configuration import... so new configs are not there yet when this install hook fires.